### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by bettybao1209**

### CODE REVIEW

**Comment by bettybao1209** [comment]
> Perhaps add the logo here will be good as well, as well as the badge for tests.

ok, so maybe it is good to add logo to each major repo.

File: README.md [hunk 1]
```diff
@@ -1,49 +1,71 @@
...
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="License">
+  </a>
+  <a href="https://github.com/neo-project/neo-modules/releases">
+    <img src="https://badge.fury.io/gh/neo-project%2Fneo-modules.svg" alt="Current neo-modules version.">
+  </a>    
+</p>
+
 ## What is it
-A set of plugins that can be used inside the NEO core library. Check [here](http://docs.neo.org/en-us/node/plugin.html) for the official documentation.
+A set of plugins that can be used inside the NEO core library are available in this repository. You can refer to [the official documentation](https://docs.neo.org/docs/en-us/node/cli/setup.html) for the more detailed usage guide. In addition, a C# SDK is included for developers to call RPC methods with ease.
```
Comment (vncoelho) [inline README.md]: `A set of plugins (...) is available`

File: README.md [hunk 2]
```diff
@@ -1,49 +1,70 @@
...
+  <a href="https://github.com/neo-project/neo-modules/releases">
+    <img src="https://badge.fury.io/gh/neo-project%2Fneo-modules.svg" alt="Current neo-modules version.">
+  </a>    
+</p>
+
 ## What is it
-A set of plugins that can be used inside the NEO core library. Check [here](https://docs.neo.org/docs/en-us/node/cli/setup.html) for the official documentation.
 
-## Using plugins
+A set of plugins that can be used inside the NEO core library is available in this repository. You can refer to [the official documentation](https://docs.neo.org/docs/en-us/node/cli/setup.html) for the more detailed usage guide. In addition, a C# SDK is included for developers to call RPC methods with ease.
```
Comment (vncoelho) [inline README.md]: I think that this " In addition, a C# SDK is included for developers to call RPC methods with ease." should come in the description of the module itself

File: README.md [hunk 3]
```diff
@@ -1,49 +1,71 @@
...
+<a href="https://neo.org/">
+      <img
+      src="https://neo3.azureedge.net/images/logo%20files-dark.svg"
+      width="250px" alt="neo-logo">
+  </a>
+</p>
+
+<p align="center">      
+  <a href="https://travis-ci.org/neo-project/neo-modules">
+    <img src="https://travis-ci.org/neo-project/neo-modules.svg?branch=master" alt="Current TravisCI build status.">
```
Comment (vncoelho) [inline README.md]: Perhaps we can leave this comment for now, I think that status is not enabled for `master` branch, correct @erikzhang?

File: README.md (hunk 2 above)
Comment (vncoelho) [reply README.md]: I slightly modified it.

**Comment by vncoelho** [comment]
@chenquanyu, `RpcClient` has dependency with `RpcNep5Tracker`, right?

**Comment by chenquanyu** [comment]
@vncoelho, `RpcClient` has the only dependency on `Neo`, but if you want to call  `Nep5Tracker` 
RPC methods, you should add `RpcNep5Tracker` plugin for the provider node.

**Comment by vncoelho** [comment]
That is what I imagined, perhaps it is good to add a brief note on it here.

**Comment by chenquanyu** [comment]
> That is what I imagined, perhaps it is good to add a brief note on it here.

In fact, `RpcServer` is also a plugin now. This is the basic requirement for the provider node😂

**Comment by vncoelho** [comment]
I mentioned that in the other thread we were discussion, right?
Then, we need to enable both of them together, correct?

**Pull request merged**